packages/mdc-select/index.js

```diff
@@ -122,6 +122,16 @@ export class MDCSelect extends MDCComponent {
       setMenuElAttr: (attr, value) => this.menuEl_.setAttribute(attr, value),
       rmMenuElAttr: (attr) => this.menuEl_.removeAttribute(attr),
       getMenuElOffsetHeight: () => this.menuEl_.offsetHeight,
+      getAdjustedMenuTop: (index) => {
+        const item = this.options[index];
+        const menu = this.menuEl_;
+        if (((item.offsetTop + item.offsetHeight) - menu.scrollTop) > menu.offsetHeight) {
```

Comment:
What is the logic behind this? It isn't clear to me why we need to do this comparison through the code. It seems like it works, but I have a feeling we are doing too much.